demo/shared/pages/home/Home.module.css

```diff
@@ -0,0 +1,26 @@
+.home {
+    min-height: 350vh;
+
+    & h1:nth-of-type(2) {
+        position: absolute;
+        top: 100vh;
+        width: 100%;
+        border-bottom: 1px solid blue;
+        color: blue;
+    }
+    & h1:nth-of-type(3) {
+        position: absolute;
+        top: 200vh;
+        width: 100%;
+        border-bottom: 1px solid blue;
+        color: blue;
+    }
+
+    & h1:nth-of-type(4) {
+        position: absolute;
+        top: 300vh;
+        width: 100%;
+        border-bottom: 1px solid blue;
+        color: blue;
+    }
```

Comment:
```suggestion
    & h1 {
        position: absolute;
        width: 100%;
        border-bottom: 1px solid blue;
        color: blue;
    }
    
    & h1:nth-of-type(2) {
        top: 100vh;
    }
    
    & h1:nth-of-type(3) {
        top: 200vh;
    }
    
    & h1:nth-of-type(4) {
        top: 300vh;
    }
```

Reply:
It makes sense but i didn't want to change the page title as well, so besides this update you suggested i changed the viewport indicators from h1 to h2.